crates/rollup-boost/src/health.rs

```diff
@@ -308,6 +333,7 @@ mod tests {
 
         let health_handle = HealthHandle {
             probes: probes.clone(),
+            execution_mode: Arc::new(Mutex::new(ExecutionMode::Enabled)),
             builder_client: builder_client.clone(),
             health_check_interval: Duration::from_secs(60),
             max_unsafe_interval: 5,
```

Comment:
Can we have another test for making sure the Health probe is 200 when ExecutionMode != Enabled and unsafe head lags more than the unsafe interval?